core/src/main/groovy/noe/ews/server/tomcat/TomcatProperties.groovy

```diff
@@ -7,9 +7,17 @@ import noe.common.utils.Library
 /**
  *
  * @author Michal Karm Babacek <mbabacek@redhat.com>
+ * @author Paul Lodge <plodge@redhat.com>
  *
  */
 class TomcatProperties {
-  public static final String TOMCAT_MAJOR_VERSION = Library.getUniversalProperty('tomcat.major.version', '7')
-  public static final String PUBLIC_IP_ADDRESS = Library.getUniversalProperty('tomcat.public.ip.address', DefaultProperties.HOST)
+  public static final String TOMCAT_MAJOR_VERSION
+  public static final String PUBLIC_IP_ADDRESS
+
+  static {
+    TOMCAT_MAJOR_VERSION = Library.getUniversalProperty('tomcat.major.version')
+    assert TOMCAT_MAJOR_VERSION?.trim() : "TOMCAT_MAJOR_VERSION is either null or empty"
```

Comment:
Imho, in our situation would be better to use `IllegalArgumentException` since assert is too general.
````
if (TOMCAT_MAJOR_VERSION?.trim()) {
  IllegalArgumentException("TOMCAT_MAJOR_VERSION is either null or empty")
}
```

Otherways LGTM.

Reply:
yes, I agree